cep-5.md

```diff
@@ -0,0 +1,52 @@
+<table>
+<tr><td> Title </td><td> Migration to Grayskull from conda-skeleton </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Mahe Iram Khan &lt;miramk163@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> March 29, 2022</td></tr>
+<tr><td> Updated </td><td> _ _, 2022</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-incubator/grayskull/issues/306 </td></tr>
+</table>
+
+## Abstract
+
+conda-skeleton, the conda recipe generator, unfortunately is not well maintained. It is also deeply integrated with conda build itself. Grayskull, a community innovation, on the other hand is a stand alone conda recipe generator. Migrating away from conda-skeleton and adopting Grayskull as the de facto conda recipe generator would allow easy maintainence and faster generation of recipes.
+
+## Other sections
+
+**Specification**
```

Comment:
This is a formatting suggestion. I suggest that instead of making the word "Specification" bold, that you instead make it a second level heading:

```
## Specification

Grayskull presently supports... 
```

This will make the document easier to read and skim over. I would suggest that you do the same for "Motivation" and "Backwards Compatibility" too.